<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
    <line>
      <![CDATA[Data:Username]]>
    </line>
    <line>
      <![CDATA[Row:1]]>
    </line>
    <line>
      <![CDATA[Test:testA is Passed & No Photo]]>
    </line>
    <line>
      <![CDATA[from test B]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="13097" started-at="2018-09-28T11:41:56Z" finished-at="2018-09-28T11:42:09Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="13097" started-at="2018-09-28T11:41:56Z" finished-at="2018-09-28T11:42:09Z">
      <class name="script.TestDemo">
        <test-method status="PASS" signature="OpenApplication(java.lang.String, java.lang.String)[pri:0, instance:script.TestDemo@5622fdf]" name="OpenApplication" is-config="true" duration-ms="10498" started-at="2018-09-28T11:41:56Z" finished-at="2018-09-28T11:42:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[localhost]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OpenApplication -->
        <test-method status="PASS" signature="testA()[pri:0, instance:script.TestDemo@5622fdf]" name="testA" duration-ms="1668" started-at="2018-09-28T11:42:07Z" finished-at="2018-09-28T11:42:08Z">
          <reporter-output>
            <line>
              <![CDATA[Data:Username]]>
            </line>
            <line>
              <![CDATA[Row:1]]>
            </line>
          </reporter-output>
        </test-method> <!-- testA -->
        <test-method status="PASS" signature="CloseApplication(org.testng.ITestResult)[pri:0, instance:script.TestDemo@5622fdf]" name="CloseApplication" is-config="true" duration-ms="870" started-at="2018-09-28T11:42:08Z" finished-at="2018-09-28T11:42:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testA status=SUCCESS method=TestDemo.testA()[pri:0, instance:script.TestDemo@5622fdf] output=Data:Username]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test:testA is Passed & No Photo]]>
            </line>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
      </class> <!-- script.TestDemo -->
      <class name="script.TestDemo2">
        <test-method status="FAIL" signature="testB()[pri:0, instance:script.TestDemo2@4883b407]" name="testB" duration-ms="10" started-at="2018-09-28T11:42:09Z" finished-at="2018-09-28T11:42:09Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at script.TestDemo2.testB(TestDemo2.java:13)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[from test B]]>
            </line>
          </reporter-output>
        </test-method> <!-- testB -->
      </class> <!-- script.TestDemo2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
